cmake/projects/getopt/hunter.cmake

```diff
@@ -0,0 +1,24 @@
+# !!! DO NOT PLACE HEADER GUARDS HERE !!!
+if(MSVC) #only for Visual Studios, all other posix compiliers should have getopt
```

Comment:
I guess it's better to do:
```cmake
if(NOT MSVC)
  hunter_user_error("Only for MSVC")
endif()
```

Usually such checks performed right before `hunter_download`